html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title> CODING FORMS </title>
+</head>
+<body>
+
+  <h1> Coding Form! </h1>
+
+  <form action="">
+
+      <label for="email"> Email </label>
+      <div>
+        <input type="text" id="email">
+      </div>
+
+      <label for="reason"> Why is coding cool? </label>
+      <div>
+        <select name="Select an option" id="reason">
+        <option value="" disabled selected> Select an option </option>
+        <option value="interesting"> Because it's interesting </option>
+        <option value="fun"> Because it's fun </option>
+        <option value="powerful"> Because it's powerful </option>
+      </select>
+      </div>
+
+      <label for="lang"> Favorite Language (choose one) </label>
+      <div>
+        <input type="radio" name="lang" value="HTML"> HTML
+        <input type="radio" name="lang" value="CSS"> CSS
+        <input type="radio" name="lang" value="JS"> Javascript
+      </div>
+
+      <label for="signup"> Would you like to sign up for more information? </label>
+      <div>
+      <input type="checkbox" id="signup"> Sign me up!
+      </div>
+
+      <label for="comments"> Comments </label>
+      <div>
+        <textarea name="textbox" id="comments" cols="30" rows="10"> </textarea>
+      </div>
+      <div>
+        <button> Submit </button>
```

Comment:
this will work because a button's type by default is `submit`, but make sure to check out the `input` with the type of submit as well!